app/assets/src/components/views/phylo_tree/PhyloTreeInputs.jsx

```diff
@@ -3,6 +3,8 @@ import React from "react";
 import { Input, Checkbox, Accordion, Menu } from "semantic-ui-react";
 import PrimaryButton from "../../ui/controls/buttons/PrimaryButton";
 
+const MIN_READS = 100;
```

Comment:
Humm, not a fan of this constraint here: for one, we need the same filter on the report page, to decide if we show the link to create the tree (and we would need to keep them in sync); also, this can be changed/bypassed by an user.

- Are we enforcing this limit on the server side too?
- Can we get this value from the server?

Reply:
For now it's purely cosmetic and not enforced on the server side. If it is not satisfied it won't break the pipeline in any way. I think it will make sense to move this constant to server side once we decide on it dynamically -- for now it's just an arbitrary threshold

Reply:
It's just a filter that helps you select relevant samples more quickly

Reply:
I'll consider defining the condition on the server side in the next iteration

Reply:
ok, but make sure you enforce it in the PipelineSampleReport too.